openapi.yaml

```diff
@@ -6726,6 +6738,50 @@ paths:
           linode-cli lke pool-create 12345 \
             --type g6-standard-4 \
             --count 6
+  /lke/clusters/{clusterId}/recycle:
+    parameters:
+    - name: clusterId
+      in: path
+      description: ID of the Kubernetes cluster to be recycled.
+      required: true
+      schema:
+        type: integer
+    x-linode-cli-command: lke
+    post:
+      operationId: postLKEClusterRecycle
+      x-linode-cli-action: cluster-recycle
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - lke:read_write
+      tags:
+      - Linode Kubernetes Engine (LKE)
+      summary: Kubernetes Cluster Recycle
+      description: |
+        Recycles a designated Kubernetes Cluster. All Linodes within the Cluster will be deleted
+        and replaced with new Linodes on a rolling basis, which may take several minutes. Replacement Nodes are
+        installed with the latest available patch for the Cluster's Kubernetes Version.
+
+        **Any local storage on deleted Linodes (such as "hostPath" and "emptyDir" volumes, or "local" PersistentVolumes) will be erased.**
+      responses:
+        '200':
+          description: Cluster has been recycled.
```

Comment:
When you get a 200 OK from this endpoint, it doesn't mean the cluster has been recycled, but rather that the request to recycle the cluster has succeeded and is now in progress.